engine/apps/slack/models/slack_message.py

```diff
@@ -103,12 +128,16 @@ def permalink(self) -> typing.Optional[str]:
 
     @property
     def deep_link(self) -> str:
-        return f"https://slack.com/app_redirect?channel={self.channel_id}&team={self.slack_team_identity.slack_id}&message={self.slack_id}"
+        return f"https://slack.com/app_redirect?channel={self.channel_slack_id}&team={self.slack_team_identity.slack_id}&message={self.slack_id}"
 
-    def send_slack_notification(self, user, alert_group, notification_policy):
+    def send_slack_notification(self, user: "User", notification_policy: "UserNotificationPolicy") -> None:
```

Comment:
given that this is an instance method, `self.alert_group` already is a thing, so I don't _think_ we need to re-pass in `alert_group`?